cc.newspring.Apollos/Utilities/NewtonsoftJsonSerializer.cs

```diff
@@ -27,7 +27,21 @@ public NewtonsoftJsonSerializer( ISerializer serializer )
 
         public string Serialize( object obj )
         {
-            var json = Newtonsoft.Json.JsonConvert.SerializeObject( obj );
+            var json = string.Empty;
+            var group = obj as Rock.Model.Group;
+
+            if ( group != null )
+            {
+                var temp = group.GroupType;
+                group.GroupType = null;
+                json = Newtonsoft.Json.JsonConvert.SerializeObject( group );
```

Comment:
When Rock posts data to Apollos because of a trigger calling APISync, OData is not used.  OData is used when getting resources is requested from a REST client.